static/app/components/replays/diff/replayDiffChooser.tsx

```diff
@@ -39,7 +39,9 @@ export default function ReplayDiffChooser({
         <TabList>
           <TabList.Item key={DiffType.SLIDER}>{t('Slider Diff')}</TabList.Item>
           <TabList.Item key={DiffType.VISUAL}>{t('Side By Side Diff')}</TabList.Item>
-          <TabList.Item key={DiffType.HTML}>{t('HTML Diff')}</TabList.Item>
+          <TabList.Item key={DiffType.HTML} hidden>
```

Comment:
Can we make this a FF, 
Or, if we want to do this fast, use a constant at the top of the file so we don't forget. Ex: IS_HIDDEN = {DiffType.HTML: true}

Reply:
Actually having the constant is kinda unnecessary since this is temporary 😅

So the question is should we have a FF or not. Up to yall

Reply:
lets not over complicate it. i think we're disabling it because of a code issue, so i think we would only re-enable it as part of a PR that fixes the code issue.

Reply:
It's also 3 lines of code, so re-enabling can be just a click on "revert PR" in GitHub